### PR TITLE
#164361985 Create article categories 

### PR DESCRIPTION
**Pull request by walimike**
**What does this PR do?**

* Create categories to which articles shall belong to up on their creation.

**Description of the tasks to be completed?**

* Create, read, update and delete of categories

**How should this be manually tested?**
* An admin user has been created on Heroku with these credentials:
 Email: admin@gmail.com
 Password: password123
* Log in at this url `api/users/login` and use the token to access the endpoints.
       - `/api/categories/`
       - `/api/categories/<str:slug>/`

**screenshots**

`api/categories` GET 
image

`api/categories` POST 
image

`api/categories/slug` GET specific 
image

`api/categories/slug` UPDATE 
image

`api/categories/slug` DELETE 
image

**What are the relevant pivot tracker stores?**
[Delivers #164361985]

### CODE REVIEW

**Review by Stanley-Okwii — APPROVED** [review]
Big ups, great work

**Review by NLSanyu — APPROVED** [review]

**Comment by akram256** [comment]
Hey, @walimike  please work on the two commits you have. it should one commit I think

File: authors/apps/articles/utils.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from django.utils.text import slugify
+
+
+def get_unique_slug(model_instance, slugable_field_name, slug_field_name):
```
Comment (akram256) [inline authors/apps/articles/utils.py:4]: Nice one

**Review by akram256 — APPROVED** [review]
Great work and changes

**Comment by walimike** [comment]
> Hey, @walimike please work on the two commits you have. it should one commit I think

Done

**Review by PatrickCmd — APPROVED** [review]

**Pull request merged**